go.mod

```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	github.com/fsnotify/fsnotify v1.4.9
+	golang.org/x/sys v0.7.0 // indirect
```

Comment:
It doesn't compile locally without this dependency update
https://stackoverflow.com/questions/71507321/go-1-18-build-error-on-mac-unix-syscall-darwin-1-13-go253-golinkname-mus